sway-lib-std/src/vm/evm/ecr.sw

```diff
@@ -13,8 +13,7 @@ pub fn ec_recover_evm_address(signature: B512, msg_hash: b256) -> Result<EvmAddr
     let pub_key_result = ec_recover(signature, msg_hash);
 
     match pub_key_result {
-        Result::Err(e) => Result::Err(e),
-        _ => {
+        Result::Err(e) => Result::Err(e), _ => {
```

Comment:
yikes, this formatting...

Reply:
I'm tired of fighting the formatter, as long as CI is happy with it. ( I know a new formatter version is coming which hopefully will clean this up).

Reply:
If you want to avoid this, you can enclose each arm in `{ }`! https://github.com/FuelLabs/sway/blob/60c626cf486d5c53c44d84db1ec81cb90174cad3/examples/wallet_smart_contract/src/main.sw#L41-L46

Reply:
I think it's better to wait till formatter v2 is out than to `{}`